types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+}
+/**
+ * This interface defines information about battery usage of application.
+ *
+ * @since 4.0
+ */
+export interface ApplicationBatteryUsage {
+    /**
+     * An attribute storing ID of an application.
+     */
+    readonly appId: ApplicationId;
+    /**
+     * An attribute which stores information about battery usage of an application with ApplicationId _appId_.
+     * Battery usage is a ratio of battery consumption of the application with ApplicationId _appId_ to the total battery consumption of all applications.
+     * The attribute value scales from 0 to 1, the higher value, the more battery is consumed.
+     */
+    readonly batteryUsage: number;
+}
+/**
+ * This interface defines the certificate information of an installed application.
+ *
+ * @since 2.0
+ */
+export interface ApplicationCertificate {
+    /**
+     * An attribute to store the type of the application certificate
+     */
+    readonly type: string;
+    /**
+     * An attribute to store the value of the application certificate
+     */
+    readonly value: string;
+}
+/**
+ * This interface defines the information available about a running
+ * application.
+ */
+export interface ApplicationContext {
+    /**
+     * An attribute to store the ID of an installed application.
+     */
+    readonly appId: ApplicationId;
+    /**
+     * An attribute to store the ID of a running application.
+     */
+    readonly id: ApplicationContextId;
+}
+/**
+ * This interface consists of an operation, URI, MIME type,
+ * and data. It describes an action to be performed by other applications
+ * and is passed to launch other applications.
+ * If the system gets the application control request, it finds
+ * the corresponding application to be launched with the delivered application control
+ * and launches the selected application.
+ *
+ * @since 2.0
+ */
+export class ApplicationControl {
+    constructor(operation: string, uri?: string | null, mime?: string | null, category?: string | null, data?: ApplicationControlData[] | null, launchMode?: ApplicationControlLaunchMode | null);
+    /**
+     * An attribute to store the category of the application to be launched.
+     */
+    category: string | null;
+    /**
+     * An array of attributes to store the data needed for an application control.
+     */
+    data: ReadonlyArray<ApplicationControlData>;
+    /**
+     * An attribute to specify launch mode. Default application launch mode is _SINGLE_.
+     *
+     * @since 2.4
+     */
+    launchMode: ApplicationControlLaunchMode;
+    /**
+     * An attribute to store the MIME type of content.
+     */
+    mime: string | null;
+    /**
+     * An attribute to store the string that defines the action to be
+     * performed by an application control.
+     */
+    operation: string;
+    /**
+     * An attribute to store the URI needed by an application control.
+     */
+    uri: string | null;
+}
+export interface ApplicationControl {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.